### PR TITLE
Inc deploy prod main microservice

### PR DESCRIPTION
**Pull request by KamenshikBackTeam**

### CODE REVIEW

**Comment by KamenshikBackTeam** [comment]
run build production

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

File: apps/main/test/jest-e2e.json [hunk 1]
```diff
@@ -1,9 +1,14 @@
 {
   "moduleFileExtensions": ["js", "json", "ts"],
-  "rootDir": ".",
+  "rootDir": "../../../.",
```
Comment (muzyk0) [inline apps/main/test/jest-e2e.json]: Давай лучше вот так

```json
{
  "moduleFileExtensions": ["js", "json", "ts"],
  "rootDir": ".",
  "testEnvironment": "node",
  "testRegex": ".e2e-spec.ts$",
  "transform": {
    "^.+\\.(t|j)s$": "ts-jest"
  },
  "moduleNameMapper": {
    "^@app/config(|/.*)$": "<rootDir>/../../../libs/config/src/$1",
    "^@app/prisma(|/.*)$": "<rootDir>/../../../libs/prisma/src/$1",
    "^@app/core(|/.*)$": "<rootDir>/../../../libs/core/src/$1"
  }
}
````

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by KamenshikBackTeam** [comment]
run build production main

**Comment by Lemnigut** [comment]
run build production main

**Comment by Lemnigut** [comment]
run build production main

**Pull request merged**